XSDElement/src/main/java/com/modelsolv/reprezen/gentemplates/xsdelement/xtend/PropertyRealizationHelper.xtend

```diff
@@ -0,0 +1,76 @@
+/*******************************************************************************
+ * Copyright © 2013, 2016 Modelsolv, Inc.
+ * All Rights Reserved.
+ *
+ * NOTICE: All information contained herein is, and remains the property
+ * of ModelSolv, Inc. See the file license.html in the root directory of
+ * this project for further information.
+ *******************************************************************************/
+
+package com.modelsolv.reprezen.gentemplates.xsdelement.xtend
+
+import com.google.common.collect.Iterables
+import com.modelsolv.reprezen.restapi.PropertyRealization
+import com.modelsolv.reprezen.restapi.ServiceDataResource
+import com.modelsolv.reprezen.restapi.TypedMessage
+import com.modelsolv.reprezen.restapi.datatypes.PrimitiveProperty
+import com.modelsolv.reprezen.restapi.datatypes.ReferenceProperty
+
+class PropertyRealizationHelper {
+	
+	extension FeatureHelper = new FeatureHelper
+
+	def private dispatch Iterable<PropertyRealization> getSequencePropRzs(ServiceDataResource dataResource) {
+		dataResource.includedProperties.getSequencePropRzs
+	}
+
+	def private dispatch Iterable<PropertyRealization> getSequencePropRzs(TypedMessage message) {
```

Comment:
Yes.  Pronounced "propers," like [Aretha Franklin](http://blog.oxforddictionaries.com/2013/03/r-e-s-p-e-c-t-find-out-what-it-means-to-me/) says.